exporter.go

```diff
@@ -40,26 +53,26 @@ func (e *Exporter) Export(
 	var err error
 
 	meta := metadata.LayersMetadata{}
-
 	meta.RunImage.TopLayer, err = workingImage.TopLayer()
 	if err != nil {
 		return errors.Wrap(err, "get run image top layer SHA")
 	}
 
+	// TODO why are we saving stack in addition to runImageRef when the two can be different?
```

Comment:
Question to the reviewers. Since metadata.Stack only contains the run image which may or may not have been used, why is it saved in addition to the actual run image in metadata.RunImage?